From 5cdfd825e68f06ad4a2703624bc59c5ff8255402 Mon Sep 17 00:00:00 2001
From: Thor Thayer <tthayer@opensource.altera.com>
Date: Fri, 22 Apr 2016 10:33:39 -0500
Subject: [PATCH 08/25] FogBugz #411698-4: socfpga: dts: Add A10-SR Devkit
 power supply alarms

commit  cec25d893ded8f9d618a1b1fb8767d46302a247b from
https://github.com/altera-opensource/linux-socfpga.git

Add the Altera Arria10 Devkit power supply alarms.
[backport 'commit c5c3b0edeffe ("FogBugz #411698-4: socfpga: dts:
                         Add A10-SR Devkit power supply alarms")']

Signed-off-by: Thor Thayer <tthayer@opensource.altera.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
index b2e19b4..2bade79e 100644
--- a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
@@ -126,6 +126,10 @@
 			compatible = "altr,a10sr-reset";
 			#reset-cells = <1>;
 		};
+
+		ps_alarm {
+			compatible = "altr,a10sr-hwmon";
+		};
 	};
 };
 
-- 
2.8.1

